ooni/report/cli.py

```diff
@@ -20,7 +20,7 @@ class Options(usage.Options):
          "Specify the configuration file to use."],
         ["collector", "c", None,
          "Specify the collector to upload the result to."],
-        ["bouncer", "b", "httpo://nkvphnp3p6agi5qq.onion",
+        ["bouncer", "b", None,
```

Comment:
Perhaps stupid question: so, this means that the default is changed, right? Has this implications?